### PR TITLE
qa/tasks/nfs: Test mounting of export created with nfs command

### PR DESCRIPTION
**Pull request by varshar16**
<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by varshar16** [comment]
Tests Passed: http://qa-proxy.ceph.com/teuthology/varsha-2020-08-24_09:41:45-rados-wip-varsha-testing-distro-basic-smithi/

File: qa/tasks/cephfs/test_nfs.py [hunk 1]
```diff
@@ -104,7 +104,7 @@ def _test_create_cluster(self):
         self._check_nfs_server_status()
         self._nfs_cmd('cluster', 'create', self.export_type, self.cluster_id)
         # Wait for few seconds as ganesha daemon take few seconds to be deployed
-        time.sleep(8)
+        time.sleep(20)
```
Comment (sebastian-philipp) [inline qa/tasks/cephfs/test_nfs.py:107]: Is there a way to something like

```python
retry_count = 0
while self._check_nfs_status() != 'running':
    retry_count += 1
    if retry_count > 20:
        self.fail('retry-count exceeded. NFS not running')
    time.sleep(5)
```

There may already be a helper-method somewhere to achieve this

**Review by sebastian-philipp — APPROVED** [review]

File: qa/tasks/cephfs/test_nfs.py (hunk 1 above)
Comment (varshar16) [reply qa/tasks/cephfs/test_nfs.py]: @sebastian-philipp Thanks!! I have updated the PR.

**Comment by varshar16** [comment]
Tests Passed: http://qa-proxy.ceph.com/teuthology/varsha-2020-08-24_13:49:08-rados-wip-test-nfs-export-mnts-distro-basic-smithi/

**Pull request merged**